lsp-methods.el

```diff
@@ -344,6 +409,48 @@ disappearing, unset all the variables related to it."
   "Client Text document capabilities according to LSP."
   `(:synchronization (:didSave t)))
 
+;;;###autoload
```

Comment:
If this function is called in the workspace hook, the autoload can be removed? As lsp-mode should already be loaded, when the hook is executed.